### DISCUSSION TITLE
能否具体说明一下如何接入说说和友链

### DISCUSSION THREAD

**Discussion by Dr-WillMos** [Qexo/Qexo#371] [Q&A] [answered]
我尝试了很久，发现无法引入qexo的js，且无法触发GIthub Action，大概率是我自己的问题，还希望大神提供帮助。

**Comment by Fgaoxing** [comment]
这个我来，Qexo没用GIthub Action，另外JS是指那个？-

**Comment by Fgaoxing** [comment]
给个网址

**Reply by Dr-WillMos** [reply]
感谢您的回复，我的网址是blog,willmo.top我按照官方文档执行相关操作，但不生效，qexo中发布的说说没有显示。
![图片](https://github.com/Qexo/Qexo/assets/72198229/f8b14514-8dea-4e2c-8e9e-4c95560aa53e)
![图片](https://github.com/Qexo/Qexo/assets/72198229/1ca03569-400c-436a-8ebe-97feaef961cb)

**Comment by Fgaoxing** [comment]
友链看这https://github.com/Qexo/Qexo-Friends/wiki



在 2023-05-13 15:46:20，"Willmo" ***@***.***> 写道：

感谢您的回复，我的网址是blog,willmo.top我按照官方文档执行相关操作，但不生效，qexo中发布的说说没有显示。

**Reply by Dr-WillMos** [reply]
我就是按照文档进行的，但是没有作用，您是否愿意告诉我相关细节以及您操作的流程？

**Reply by am-abudu** [reply]
我希望可以看到你的网页是如何的

**Reply by Dr-WillMos** [reply]
![image](https://github.com/Qexo/Qexo/assets/72198229/a6f1969b-89b0-43ae-a8c4-ab5a542f57ff)
talks文件夹下的md文件如下，期待您的回复。

**Reply by am-abudu** [reply]
请去掉${}

**Reply by Dr-WillMos** [reply]
![image](https://github.com/Qexo/Qexo/assets/72198229/8c0df08f-0dc1-415b-8e20-a457c4076a64)
已经去掉了，但还是没有显示。

**Reply by am-abudu** [reply]
我进入你的页面后发现js并未被导入，请检查

**Reply by Dr-WillMos** [reply]
![image](https://github.com/Qexo/Qexo/assets/72198229/92f22cd6-2a0d-4192-9470-9b9bb4b73120)
![image](https://github.com/Qexo/Qexo/assets/72198229/60f67589-e0ee-42e0-a7ad-485861edd778)
我已经引入了js，但没有显示，是我引入的方法不对吗，我应该怎么做

**Reply by am-abudu** [reply]
我没有使用过你的主题，但你或许可以尝试使用{%raw%} {%endraw%}包裹整段代码

**Reply by Dr-WillMos** [reply]
![image](https://github.com/Qexo/Qexo/assets/72198229/a3284f6a-634a-4ded-9bad-14f28b65b928)
有点不理解你说的，请问是这样吗？

**Reply by am-abudu** [reply]
是的

**Reply by Dr-WillMos** [reply]
![image](https://github.com/Qexo/Qexo/assets/72198229/36135a41-b126-4389-b262-9f712f5f5c55)
我尝试了，但没有效果。。。

**Reply by Dr-WillMos** [reply]
请问您有解决此问题的打算吗？

**Reply by am-abudu** [reply]
我依然认为这不是我的问题。因为我的blog: www.oplog.cn 是正常的，并且我无法复现你的问题，或许你的博客有什么特别的设置?我无法解决不能确定的问题。若是要解决，我需要你的博客源码的完整拷贝并在有时间的时候逐一排查问题

**Reply by dishao123** [reply]
我知道这个问题，应该是跟主题自带了友链功能有关

**Reply by am-abudu** [reply]
> 我知道这个问题，应该是跟主题自带了友链功能有关

但他并未使用自带友联的模板

**Reply by Dr-WillMos** [reply]
出于我的设置问题，他无法运作，您是否愿意提供联系方式（邮箱等）以便我将整个源码发给您以便在你有时间的时候排查？

**Reply by am-abudu** [reply]
可以添加我的QQ：2753534031

**Reply by dishao123** [reply]
> > 我知道这个问题，应该是跟主题自带了友链功能有关
> 
> 但他并未使用自带友联的模板

我使用了相同的主题也有一模一样的问题，不知道怎么解决

**Reply by Dr-WillMos** [reply]
> 我知道这个问题，应该是跟主题自带了友链功能有关

我的主题确实有友链功能，但我想应该不影响说说的使用。

**Reply by dishao123** [reply]
> > 我知道这个问题，应该是跟主题自带了友链功能有关
> 
> 我的主题确实有友链功能，但我想应该不影响说说的使用。

我也在用这个主题但是说说功能正常

**Reply by Dr-WillMos** [reply]
> 可以添加我的QQ：2753534031

源码已经发到您的邮箱了，感谢。

**Reply by Dr-WillMos** [reply]
> > > 我知道这个问题，应该是跟主题自带了友链功能有关
> > 
> > 
> > 我的主题确实有友链功能，但我想应该不影响说说的使用。
> 
> 我也在用这个主题但是说说功能正常

您是如何引入js的？
方便截图吗？

**Reply by dishao123** [reply]
> > > > 我知道这个问题，应该是跟主题自带了友链功能有关
> > > 
> > > 
> > > 我的主题确实有友链功能，但我想应该不影响说说的使用。
> > 
> > 
> > 我也在用这个主题但是说说功能正常
> 
> 您是如何引入js的？ 方便截图吗？

```
---
comment: waline
date: '2023-05-31T23:58:40+08:00'
title: 说说
updated: 2023-6-2T22:16:58.422+8:0
---
<head>
  <!-- ... -->
  <script src="https://cdn.jsdelivr.net/gh/Uyoahz26/daodao@main/dist/qexo-dao.min.js"></script>
  <!-- ... -->
</head>
<body>
  <!-- ... -->
  <div id="qexoDaoDao"></div>
  <script>
    qexoDaodao?.init({
      el: "#qexoDaoDao",
      avatar: "https://q1.qlogo.cn/g?b=qq&nk=2496091142&s=640",
      title: "说说",
      name: "小骏",
      limit: 10,
      useLoadingImg: false,
      baseURL: "后台url",
    }).then(function (){
      console.log("qexoDaodao加载完成");
    })
  </script>
</body>
```
这是我的说说页源码

**Reply by Dr-WillMos** [reply]
能否告诉我您是如何引入美化方案中的JS的？

**Comment by dishao123** [comment]
我也是一样的问题

**Comment by Dr-WillMos** [comment, marked as answer]
问题解决：您需要关闭主题自带的友链配置，一般是将相关配置改为flase状态
